src/other/Table.js

```diff
@@ -178,6 +186,20 @@
         ;
         th.order();
 
+        //Disable Pagination if a widget is in the data
+        if (context.pagination()) {
+            for (var iDx1 = 0; iDx1 < this._db._data.length; iDx1 ++) {
+                var d = this._db._data[iDx1];
+                for (var iDx2 = 0; iDx2 < d.length; iDx2 ++) {
+                    if (d[iDx2] instanceof Widget) {
+                        context.pagination(false);
+                        break;
+                    }
+                }
+                if (context.pagination() === false) {break;}
+            }
+        }
```

Comment:
As we already discussed, pagination obviously breaks.  For now I just test the data to see if it includes a widget, then disable pagination if it does.  I will add a separate issue to to work on the pagination, as that will be a task in itself. FYI, I believe all the other features (fixed headers, totals, formatting etc.) all work at this point.